.github/workflows/ci-java.yml

```diff
@@ -28,12 +29,9 @@ jobs:
         uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
-          java-version: '11'
+          java-version: '${{ matrix.version }}'
```

Comment:
I think I need to separate the compile and test phases here because this should bring the older JDK as well right? We just want to test on different JREs